### ISSUE TITLE
Allow watching multiple roots

### ISSUE THREAD

**Issue by kosta** [watchexec/watchexec#860]
**Is your feature request related to a problem? Please describe.**
I work with a monorepo and the build system is not very smart about detecting changes. I would like to watch for changes in every directory and run e.g. `make` in every directory that changed. Then I can just do my work and everything builds in the background "as needed".

**Describe the solution you'd like**
Possible solution: Allow specifying multiple `roots` (I think watchman does this?) and run the command inside that root.

**Describe alternatives you've considered**
1. Using splitting `$WATCHEXEC_COMMON_PATH`, getting the first directory from that, sort|uniq (so we run for each directory only once), and running the command for each such directory.
2. Use xargs to run watchexec for every subpath (but it's not made for long-running tasks such as watchexec - you can work around with parallelism though)

Side note: I tried to do this but could not get the env variable to work, do you have an example that does e.g. `echo $WATCHEXEC_COMMON_PATH` ?

**If proposing a new CLI option, option names you think would fit.**
Maybe `--root`, which could be specified multiple times? Or, was we already have `-w`, just `--run-inside-root` which would make the command run inside the watched root instead of the invoked directory.

**Additional context**

**Comment by passcod** [comment]
No, this is out of scope. Make a script that runs in each folder you desire. Your build/task system should handle that; consider [just](https://just.systems/) or [checkexec](https://github.com/kurtbuilds/checkexec). Also please read the help for `--emit-events-to`.
